README.md

```diff
@@ -12,6 +12,18 @@ To start the application run the BattleshipApplication(./src/main/java/com.game.
 
 or run in te terminal:
 
+ Battleship game built with Java 11 and Spring Boot 2.5.6
+
+## Installation
```

Comment:
Ok looks like you have a bit of duplication here. Check the raw file, looks like lines 5-14 need to be deleted.

Reply:
Done!